frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -476,6 +478,14 @@ export const managePipelineResources = async (
       docker.dockerfilePath,
       image.tag,
     );
+    pipelineResources.push(managedPipeline);
```

Comment:
`managedPipeline` can be undefined right if above if and else if is not truthy so here shall we have check?

```suggestion
    managedPipeline && pipelineResources.push(managedPipeline);
```

Reply:
managePipelineResources is being called when a pipeline is enabled, so managedPipeline here will be always set to either create flow or update flow.